src/EjectionManager.sol

```diff
@@ -60,23 +60,20 @@ contract EjectionManager is IEjectionManager, OwnableUpgradeable{
     /**
      * @notice Ejects operators from the AVSs RegistryCoordinator under a ratelimit
      * @param _operatorIds The ids of the operators 'j' to eject for each quorum 'i'
-     * @return ejectedOperatorsForQuorum The total number of operators ejected for each quorum 'i'
-     * @dev This function will eject as many operators as possible without reverting prioritizing operators at the lower index
+     * @dev This function will eject as many operators as possible prioritizing operators at the lower index
      * @dev The owner can eject operators without recording of stake ejection
      */
-    function ejectOperators(bytes32[][] memory _operatorIds) external returns (uint32[] memory){
+    function ejectOperators(bytes32[][] memory _operatorIds) external {
```

Comment:
wdyt about refactoring the ejection logic 

1. That doesn't enforce the rate limit that is callable by the owner
2. That is callable by the ejector and enforces the rate limit

This should clean up some of the branchy conditionals in this function